mautic/mautic
# Add log display limit for webhook detail page

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | 
| BC breaks?                             | 
| Automated tests included?              | yes
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
if Webhook has too many logs in millions then it detail page was showing 500 PHP error because It returns more data than php can handle. So here we are adding limit to display logs. We will display only latest 100 logs on detail page.


#### Steps to reproduce this PR:

1. Create a webhook
2. Create 2-3 millions records for log
3. Now open webhook detail page
4. Page would display 500 error

#### Steps to test this PR:

1. Create a webhook
2. Create 2-3 millions records for log
3. Now open webhook detail page
4. Page should be open and should display only latest 100 logs.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10257"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
